src-json/define.json

```diff
@@ -351,10 +351,16 @@
 		"doc": "<version:20-45> Sets the .NET version to be targeted",
 		"platforms": ["cs"]
 	},
+	{
+		"name": "NetCoreVer",
+		"define": "netcore_ver",
```

Comment:
Wouldn't `net_core_ver` be more consistent?

Reply:
Dunno, I've seen "netcore" used quite often too

Reply:
I mostly mean consistenty with the line above, where it's treated as two separate words.

Reply:
Yeah, could rename it to `NetcoreVer`. Not sure which one I should rename though.